libraries/loader/src/loader.js

```diff
@@ -158,9 +157,16 @@ function loader(componentDirectory, virtualComponents = {}) {
       }
       return loaded[target];
     }
-
-    return load(target);
+    return recursiveLoad(target);
+  };
+  //method to crash the process if error occured
```

Comment:
rather than document here, it'd be best to include this documentation in `libraries/loader/README.md` where the rest of this library is documented.  Note, too, the commenting style in the rest of this file: whitespace both before and after `//`.